Sources/Quick/QuickTestObservation.swift

```diff
@@ -45,24 +45,18 @@ extension QuickSpec {
         if let subclasses = subclasses {
             subjects = subclasses
         } else {
-            let classesCount = objc_getClassList(nil, 0)
+            // See https://developer.apple.com/forums/thread/700770.
+            var classesCount: UInt32 = 0
+            guard let classList = objc_copyClassList(&classesCount) else { return }
+            defer { free(UnsafeMutableRawPointer(classList)) }
+            let classes = UnsafeBufferPointer(start: classList, count: Int(classesCount))
```

Comment:
Could we extract this into a helper function and reuse it?

Reply:
Done.